src/GraphQL.Tests/Shared/JsonStringExtensions.cs

```diff
@@ -23,14 +21,13 @@ public static ExecutionResult ToExecutionResult(this string? json, ExecutionErro
             Executed = executed
         };
 
+    [return: NotNullIfNotNull("json")]
     public static Dictionary<string, object?>? ToDictionary(this string? json)
     {
         if (json == null)
             return null;
 
         var ret = json.ToInputs();
-        if (ret == null)
-            return null;
```

Comment:
Removed condition is impossible to occur.  `ToInputs` returns Inputs.Empty if `json` was null, and `json` has already been verified to be not-null.